applications/Web3Box.md

```diff
@@ -0,0 +1,142 @@
+# W3F Open Grant Proposal
+
+- **Project Name:** Web3Box
+- **Team Name:** Web3Box Labs
+- **Payment Address:** 0x2755213c8435F15D9929101357686a9C1Bec280A (USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview
+
+### Overview
+
+**Web3Box is committed to creating a Web3 UGA platform with a secure environment worldwide**
+
+- In order to prevent inevitable frauds, we propose a privacy-safe sandbox concept, Web3Box, so that all the information of Polkadot ecosystem users is controlled in their local trusted environment.
+- Web3Box will become the Web3 portal for Polkadot ecosystem users, giving them one-stop access to all Web3 applications. Polkadot ecosystem users can manage their multi-chain assets and track asset changes, ensuring asset security.
+- In response to the frequent Web3 asset security incidents,  Polkadot ecosystem users can track the risks of Polkadot ecosystem projects through the Web3Box, so as to choose Web3 portfolio safely.
+
+
+
+
+### Project Details
+
+#### Web3Box Architecture ####
+**![](https://lh3.googleusercontent.com/tlealryMkOZD0i7dK9LcU_TTsbO5J_13YcgeOl8hWI56sLVtnyYEDDY-dCeYLVa7_i4k4PIdHCkNqTjA5BFtqYKBJJMUh5jkklAxK9-9PKljwMfoq2a0l-WoMXgTRToE5Pih0m2qGXUoi_Yjl7dSNIo3J_3uAGBW4qaVxN3TUGKjQ88VxuhntBRcddz6JA)**
```

Comment:
1. What are the white and green lines? 
2. What feed? What tools?

Reply:
Q1:
White lines mean three key modules, Privacy-safe Box, Risk Assessment and Web3 Tool/Content Aggregation.
Green lines mean that,  In addition to the dashboard & wallet, the derivative functions of Privacy-safe box will be developed by the follow-up milestone.

Q2:
Tools mean web3 tools navigation, including mirror, medium, mask, etc.
<img width="605" alt="image" src="https://user-images.githubusercontent.com/117150831/200759211-e5dcaec2-757a-4f69-b4e7-7a6cec4b6e87.png">

Feeds mean web3 content aggregation, like RSS 3, Coindesk, DeFiLlama, Messari, etc.
<img width="599" alt="image" src="https://user-images.githubusercontent.com/117150831/200759293-603512b0-7416-4300-98ae-8a101c5b10e4.png">